manifests/config.pp

```diff
@@ -1,12 +1,16 @@
-class cerebro::config (
-  $secret              = $::cerebro::secret,
-  $hosts               = $::cerebro::hosts,
-  $basepath            = $::cerebro::basepath,
-  $java_home           = $::cerebro::java_home,
-  $java_opts           = $::cerebro::java_opts,
-  $basic_auth_settings = $::cerebro::basic_auth_settings,
-  $sysconfig           = $::cerebro::sysconfig,
-) {
+# @summary Configures Cerebro
+#
+# @api private
+class cerebro::config
+{
+  $secret              = $cerebro::secret
```

Comment:
Since the class is private and only ever expected to be declared from the base class, I've moved the variable declarations from being class parameters to being in the body of the class.